ui/app/src/views/projects/dashboards/HelperDashboardView.tsx

```diff
@@ -82,39 +83,41 @@ export function HelperDashboardView(props: GenericDashboardViewProps): ReactElem
       }}
     >
       <ErrorBoundary FallbackComponent={ErrorAlert}>
-        <PluginRegistry
-          pluginLoader={remotePluginLoader()}
-          defaultPluginKinds={{
-            Panel: 'TimeSeriesChart',
-            TimeSeriesQuery: 'PrometheusTimeSeriesQuery',
-          }}
-        >
-          <ValidationProvider>
-            <ErrorBoundary FallbackComponent={ErrorAlert}>
-              <UsageMetricsProvider project={project.metadata.name} dashboard={dashboardResource.metadata.name}>
-                <ViewDashboard
-                  dashboardResource={dashboardResource}
-                  datasourceApi={datasourceApi}
-                  externalVariableDefinitions={externalVariableDefinitions}
-                  dashboardTitleComponent={
-                    <ProjectBreadcrumbs dashboardName={getResourceDisplayName(dashboardResource)} project={project} />
-                  }
-                  emptyDashboardProps={{
-                    additionalText: 'In order to save this dashboard, you need to add at least one panel!',
-                  }}
-                  onSave={onSave}
-                  onDiscard={onDiscard}
-                  initialVariableIsSticky={true}
-                  isReadonly={isReadonly}
-                  isVariableEnabled={isLocalVariableEnabled}
-                  isDatasourceEnabled={isLocalDatasourceEnabled}
-                  isEditing={isEditing}
-                  isCreating={isCreating}
-                />
-              </UsageMetricsProvider>
-            </ErrorBoundary>
-          </ValidationProvider>
-        </PluginRegistry>
+        <ReactRouterProvider>
```

Comment:
Not sure at what level in the hierarchy I should add it, I added it close to the `PluginRegistry` for now because it's related to the panel plugins. Alternative I could also add it to `app/src/render-app.tsx` or somewhere else.

Reply:
probably it will be better to add it on the application, as the `HelperDashboardView` could be used directly in a Host application, if we leave it here then we are locking the Provider for users who want to embed a whole dashboard rather than specific panels.

Reply:
great point! I'll move it.